test/Microsoft.AspNet.Razor.Test/Utils/TestFile.cs

```diff
@@ -27,7 +27,9 @@ public static TestFile Create(string localResourceName)
 
         public Stream OpenRead()
         {
-            var strm = Assembly.GetManifestResourceStream(ResourceName);
+            string resourceName = Assembly.GetName().Name + "." + ResourceName.Replace('/', '.');
+
+            var strm = Assembly.GetManifestResourceStream(resourceName);
             if (strm == null)
             {
                 Assert.True(false, string.Format("Manifest resource: {0} not found", ResourceName));
```

Comment:
This wont be needed once you do https://github.com/aspnet/Razor/pull/351/files#r28018029